src/libs/actions/IOU.ts

```diff
@@ -1140,7 +1140,8 @@ function getUpdateMoneyRequestParams(
                 ? IOUUtils.updateIOUOwnerAndTotal(iouReport, updatedReportAction.actorAccountID ?? -1, diff, TransactionUtils.getCurrency(transaction), false, true)
                 : {};
         }
-        updatedMoneyRequestReport.cachedTotal = CurrencyUtils.convertToDisplayString(updatedMoneyRequestReport.total, updatedTransaction?.modifiedCurrency);
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        updatedMoneyRequestReport.cachedTotal = CurrencyUtils.convertToDisplayString(updatedMoneyRequestReport.total, updatedTransaction?.modifiedCurrency || updatedTransaction?.currency);
```

Comment:
Do we really need the fallback? It looks like `updatedTransaction.modifiedCurrency` gets set to the same value as `updatedTransaction.currency` in `getUpdatedTransaction`

Reply:
We need the fallback, bcoz modifiedCurrency and currency hold different values for the transaction data.

<img width="1167" alt="Screenshot 2024-02-20 at 23 35 16" src="https://github.com/Expensify/App/assets/14358475/123e5298-bdcb-4e72-a44d-b785a2c26141">

I think we can use `transactionDetails.currency` also as it is doing the same thing.
https://github.com/Expensify/App/blob/12fc2cbae18ae6fbeda22bca5b8bcd2da63f5906/src/libs/actions/IOU.ts#L1051-L1052
https://github.com/Expensify/App/blob/299576fea9e0ff0e10049dd9ce6c3912a49a7b14/src/libs/TransactionUtils.ts#L291-L297

Reply:
In `getUpdatedTransaction`, `modifiedCurrency` gets set from the `transactionChanges` whereas the original transaction `currency` remains the old value only.